build.sbt

```diff
@@ -165,7 +157,7 @@ lazy val docSettings = Seq(
         }
       s"https://github.com/freechipsproject/firrtl/tree/$branch€{FILE_PATH}.scala"
     }
-  ) ++ scalacOptionsVersion(scalaVersion.value) ++ scalacDocOptionsVersion(scalaVersion.value)
```

Comment:
maybe unrelated too?

Reply:
I removed the `-Xsource` flag in https://github.com/freechipsproject/firrtl/pull/1796/commits/8870b5cc76973e596f1a7c87a4063a3049b858bb as it should not be required for `firrtl` which does not contain any Chisel bundles.